src/SFA.DAS.Reservations.Api.UnitTests/Controllers/WhenCallingGetTrainingCourseList.cs

```diff
@@ -36,7 +37,10 @@ public class WhenCallingGetTrainingCourseList
             controllerResult.StatusCode.Should().Be((int)HttpStatusCode.OK);
             var model = controllerResult.Value as GetTrainingCoursesListResponse;
             Assert.IsNotNull(model);
-            model.Standards.Should().BeEquivalentTo(mediatorResult.Courses);
+            model.Standards.Should().BeEquivalentTo(mediatorResult.Courses, options => options
+                .Excluding(s => s.StandardUId)
+                .Excluding(s => s.LarsCode));
+            model.Standards.All(m => m.Id > 0).Should().BeTrue();
```

Comment:
if there's already a test covering the casting of types, just assert that the casts are equivalent, then you don't have to worry about all Id > 0 (they could all be 1 and pass this). e.g. 
`model.Courses.Should().BeEquivalentTo(mediatorResult.Courses.Select(course => (TypeCastTo)course)`

Reply:
Fixed